spec/lib/rwj_reporter_spec.rb

```diff
@@ -51,17 +51,17 @@
     it 'gets the right channel counts for a set of log files in the given date range' do
       rr = RwjReporter.new 'spec/fixtures/'
       rr.channel_counts('160228', '160303')
-      expect(rr.send(:channel1_counts)).to include("115b"=>1, "019a"=>1, "260"=>1, "101"=>1, "350"=>1, "336"=>1)
-      expect(rr.send(:channel2_counts)).to include("140"=>1, "118"=>1, "348"=>1, "250"=>1)
+      expect(rr.send(:channel1_counts)).to include("115b" => 1, "019a" => 1, "260" => 1, "101" => 1, "350" => 1, "336" => 1)
+      expect(rr.send(:channel2_counts)).to include("140" => 1, "118" => 1, "348" => 1, "250" => 1)
     end
   end
 
   context '#print_channel_counts_files' do
-    before(:context) {
+    before(:context) do
       RwjReporter.print_reports_for_dates('160228', '160303', 'spec/fixtures/', '.')
-    }
```

Comment:
just FYI @jmartin-sul   - the brackets are okay if it's a oneliner, which it *now* is.